weave/monitoring2/__init__.py

```diff
@@ -0,0 +1,5 @@
+__all__ = ["openai"]
+
+from . import openai
+
+# openai.patch()
```

Comment:
nit: remove

Reply:
This is a design decision -- should we auto-patch on import?